test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -260,6 +260,30 @@ public void ElementCurveReferences_ExtractsExpectedReferences()
 
         #endregion
 
+        #region Pin settings
+        private static void GetExpectedElementPinStatus(Element element, bool value)
```

Comment:
Method & variable names a bit more suggestive.
```suggestion
        private static void AssertElementPinStatusIs(Element element, bool expectedValue)
```